src/apps/components/AllAppList/AllAppList.tsx

```diff
@@ -20,18 +19,24 @@ const AllAppList: React.FC<AllAppListProps> = ({
   navigateToAppInstallPage,
   navigateToGithubForkPage,
 }) => {
-  const appsPairs = React.useMemo(() => chunk(appList, 2), [appList]);
+  const { themeValues } = useTheme();
 
   if (!appList) {
     return <Skeleton />;
   }
 
   return (
-    <Box display="flex" flexDirection="column" gap={5} marginTop={5}>
-      {appsPairs.map(appPair => (
+    <Box
+      display="grid"
+      __gridTemplateColumns="repeat(2, 1fr)"
+      __gap={`${themeValues.spacing[8]} ${themeValues.spacing[5]}`}
+      padding={5}
+      marginTop={5}
+    >
+      {appList.map(app => (
         <AppListRow
-          key={appPair[0].name.en}
-          appPair={appPair}
+          key={app.name.en}
```

Comment:
I think we allow multiple apps with the same name - maybe we can add `app.name.en-app.id`?